src/backend/users/tests.py

```diff
@@ -7,7 +7,7 @@ class AdminSiteTest(TestCase):
     @classmethod
     def setUpClass(cls):
         cls.counselor, cls.houseseeker = Client(), Client()
-        call_command("create_groups")
+        call_command("loaddata", "users/fixtures/group_permissions")
```

Comment:
This can be even simpler now that you're using fixtures: https://docs.djangoproject.com/en/3.2/topics/testing/tools/#fixture-loading

Reply:
Great, just fixed!